doc/rules/class_notation/final_internal_class.rst

```diff
@@ -19,7 +19,7 @@ Configuration
 ~~~~~~~~~~~~~~~~~~~~~~
 
 Class level annotations tags that must be set in order to fix the class. (case
-insensitive)
+insensitive).
```

Comment:
This looks odd.

Reply:
I hope you meant the dot before opening parenthesis.